docs/spec/normative_examples.md

```diff
@@ -679,12 +690,50 @@ status:
 ```
 
 Even when ready, the new revision does not automatically start serving
-traffic, as the route was pinned to revision `def`.
+production traffic, as the route was pinned to revision `def`. It will, however,
+be optionally accessible under the name `latest.my-service...` for initial
+manual testing.
 
-Once the new revision is ready, the route will update the `next` name
-to point to the revision `ghi`. The new revision will still not
-receive any traffic by default, but can be accessed for testing,
-verification, etc.
+The user can then begin the rollout of revision ghi:
+
+```http
+PATCH /apis/serving.knative.dev/v1alpha1/namespaces/default/services/my-service
+```
+```yaml
+apiVersion: serving.knative.dev/v1alpha1
+kind: Service
+metadata:
+  name: my-service
+spec:
+  release:
+    revisions: [def, ghi]
+    rolloutPercent: 0
+```
+
+This makes the route update the `next` name to point to the revision `ghi`. (The
+list of revisions can contain one or two items. If two, the first is `current`
+and the latter is `next`) The new revision will still not receive any traffic,
+but can be accessed for testing, verification, etc under the
+`next.my-service...` name.
+
+An admission hook prevents changing `revisions` without also setting
```

Comment:
I don't think we should prevent this.

It's possible to completely overwrite the revisions array and effect a sharp traffic change.

Reply:
Updated.